compass/ocean/tests/utility/create_salinity_restoring/__init__.py

```diff
@@ -0,0 +1,25 @@
+from compass.ocean.tests.utility.create_salinity_restoring.salinity_restoring import Salinity
+from compass.ocean.tests.utility.create_salinity_restoring.extrap_salin import ExtrapSalin
+from compass.testcase import TestCase
+
+
+class CreateSalinRestoring(TestCase):
```

Comment:
If this is the name of the class, please name the directory `create_salin_restoring` to match.